### ISSUE TITLE
App crashes after adb command sent.

### ISSUE THREAD

**Issue by zi-l** [amotzte/android-mock-location-for-development#5]
Test in Android 8.0, android 6.0.1 with v0.4
Seconds later after adb command is sent, app stops running and crashes.
prompt message in cmd: 
    Broadcasting: Intent { act=send.mock (has extras) }
    Broadcast completed: result=0

**Comment by amotzte** [comment]
Hi @zi-l 
Can you send stacktrace ?

**Comment by zi-l** [comment]
@amotzte No longer crash after selected as Mock Location App at Developer options page and granted location permission. Still could not set location successfully, however.  Performing the same as issue:[#2](https://github.com/amotzte/android-mock-location-for-development/issues/2)

**Comment by zi-l** [comment]
@amotzte Here's the adb log: [mockapp.log](https://gist.github.com/zi-l/b29743ab144b70c29c0b552334061dbf)

- Testbed: Galaxy S8 (Android 8.0.0)
- Release: v0.3

**Comment by amotzte** [comment]
@zi-l is mock location enabled in ```settings -> system -> Developer options -> select mock location app``` ?

**Comment by zi-l** [comment]
@amotzte Yes, and also granted app location permission. However, it does not work. The ADB command seems to be no longer effective.
